phpBB/phpbb/db/migration/data/v32x/user_notifications_table_reduce_column_sizes.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v32x;
+
+class user_notifications_table_reduce_column_sizes extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return array(
+			'\phpbb\db\migration\data\v32x\user_notifications_table_remove_duplicates',
+		);
+	}
+
+	public function update_schema()
+	{
+		return array(
+			'change_columns'	=> array(
+				$this->table_prefix . 'user_notifications'			=> array(
+					'item_type'		=> array('VCHAR:155', ''),
```

Comment:
I went ahead and changed it to 165 since it's not much of a difference from 155. I did try 200 but turned out to be too high, kind of validating the calculations:
https://travis-ci.org/phpbb/phpbb/jobs/236011337#L671